rust_src/src/fonts.rs

```diff
@@ -167,6 +170,48 @@ impl From<LispObject> for Option<LispFontObjectRef> {
     }
 }
 
+pub type LispFontSpecRef = ExternalPtr<Lisp_Font_Spec>;
```

Comment:
This could just be called `FontSpecRef`, since there's no potential for confusion.

Reply:
I prefer to keep it as it as, as it keeps the naming consistent. I think the potential confusion over different naming styles outweighs the convenience of the slightly shorter name.